Cargo.toml

```diff
@@ -74,7 +74,7 @@ datafusion-substrait = { git = "https://github.com/waynexia/arrow-datafusion.git
 etcd-client = "0.11"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/MichaelScofield/greptime-proto.git", rev = "71c0002045f6cc8d6e42609bcc5ca6cea963baad" }
+greptime-proto = { git = "https://github.com/DevilExileSu/greptime-proto.git", rev = "c7fea8059be6c5e0b1745d12b0c8304dfde5118d" }
```

Comment:
Could you please also port this to [GreptimeTeam/greptime-proto](https://github.com/GreptimeTeam/greptime-proto)?

Reply:
You can create a PR in [GreptimeTeam/greptime-proto](https://github.com/GreptimeTeam/greptime-proto), and just change the `rev` to your PR's head commit without changing the `git`.